app/controllers/thredded/topics_controller.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 module Thredded
-  class TopicsController < Thredded::ApplicationController
+  class TopicsController < Thredded::ApplicationController # rubocop:disable Metrics/ClassLength
```

Comment:
InstanceVariableAssumption: Thredded::TopicsController assumes too much for instance variable '@category'<br>InstanceVariableAssumption: Thredded::TopicsController assumes too much for instance variable '@edit_topic'<br>InstanceVariableAssumption: Thredded::TopicsController assumes too much for instance variable '@new_topic'<br>InstanceVariableAssumption: Thredded::TopicsController assumes too much for instance variable '@query'<br>IrresponsibleModule: Thredded::TopicsController has no descriptive comment<br>TooManyInstanceVariables: Thredded::TopicsController has at least 7 instance variables<br>TooManyMethods: Thredded::TopicsController has at least 16 methods